### DISCUSSION TITLE
Is there a way to have Bravo start on a certain day for Fiscal 4-5-4 and only count week days?

### DISCUSSION THREAD

**Discussion by tylercook10811** [sql-bi/Bravo#432] [Q&A] [answered]
Sorry if this is the wrong place to ask. I am kinda stuck and would love any help! Great tool so far!

**Comment by marcosqlbi** [comment, marked as answer]
You can use the Weekly calendar for that. The table always includes all the dates in the year, but you can filter the weekdays in your reports.
